src/internal/_get.js

```diff
@@ -3,9 +3,7 @@ import { Iterable } from 'immutable';
 
 const empty = () => undefined;
 
-get.implement(Array, (key, subject) => subject[key]);
```

Comment:
We no longer need to implement lookups on Array or Object anymore, as their implementations are the same as the fallback